webpages/styles/components/buttons.css

```diff
@@ -1,7 +1,7 @@
 .large-button {
   color: var(--content-text);
   background: var(--button-background);
-  border: 1px solid var(--control-border);
+  border: none;
```

Comment:
I think the border should stay on the buttons.

Reply:
the box shadow works as the border

Reply:
There are still some places where buttons are directly on the page background which causes poor contrast but now also without a border.

Reply:
Also a couple buttons used to be highlighted yellow such as the confirm button when importing settings.